### PR TITLE
MNT Use gha-dispatch-ci

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripeltd/product-issues/issues/698

This was an automatically generated pull-request - peer reviewing that we're happy for a large number of modules to have pull-requests like this one

Generated using https://github.com/emteknetnz/module-standardiser

### CODE REVIEW

File: .github/workflows/dispatch-ci.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Dispatch CI
+
+on:
+  # At 03:00 PM UTC, only on Sunday and Monday
```
Comment (GuySartorelli) [inline .github/workflows/dispatch-ci.yml:4]: 03:00 would be AM

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -4,9 +4,6 @@ on:
   push:
   pull_request:
   workflow_dispatch:
-  # Every Sunday at 3:00pm UTC
-  schedule:
```
Comment (GuySartorelli) [inline .github/workflows/ci.yml:8]: Also remove the if condition below.

Comment (emteknetnz) [reply .github/workflows/ci.yml:8]: Updated

File: .github/workflows/dispatch-ci.yml (hunk 1 above)
Comment (emteknetnz) [reply .github/workflows/dispatch-ci.yml]: Updated

**Review by GuySartorelli — APPROVED** [review]
Looks good, not merging because it's draft.

**Comment by GuySartorelli** [comment]
PHPlint failure is unrelated

**Pull request merged**